src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -493,6 +488,12 @@ class EmojiPickerMenu extends Component {
                         autoCorrect={false}
                     />
                 </View>
+                {!isFiltered && (
+                    <CategoryShortcutBar
+                        headerIndices={this.headerIndices}
+                        onPress={this.scrollToHeader}
+                    />
+                )}
```

Comment:
@stitesExpensify  If you're good with [using absolute positioning](https://github.com/Expensify/App/pull/14652#discussion_r1095972455) for the FlatList with the search items, I can remove the `isFiltered` option.